packages/flutter/lib/src/material/material_button.dart

```diff
@@ -457,9 +457,13 @@ class MaterialButton extends StatelessWidget {
   }
 }
 
-/// The type of [MaterialButton]s created with [RaisedButton.icon], [FlatButton.icon],
-/// and [OutlineButton.icon].
+/// The type of [MaterialButton]s created with RaisedButton.icon, FlatButton.icon,
+/// and OutlineButton.icon.
 ///
 /// This mixin only exists to give the "label and icon" button widgets a distinct
 /// type for the sake of [ButtonTheme].
+@Deprecated(
+  'This is no longer used by the framework aside from deprecated API. '
```

Comment:
("APIs"?)

Is this message one that will make sense when people who use this see it? Like, what should they replace it with? Or is that question not meaningful? (I'm not familiar with this API.)